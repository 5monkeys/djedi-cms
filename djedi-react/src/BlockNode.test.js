import React from "react";
import renderer from "react-test-renderer";

import { BlockNode } from ".";

test("it works", () => {
  const component = renderer.create(<BlockNode />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
