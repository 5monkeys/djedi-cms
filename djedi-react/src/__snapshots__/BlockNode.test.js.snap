// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
<span>
  BlockNode
</span>
`;
